src/components/FAQPage.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <div class="pt-4">
+    <QuestionAndAnswer question="What is RateMyCoop?"
+                       answer="RateMyCoop is a student created website allowing students to leave a review on a company after they have completed a coop work term."/>
```

Comment:
Here and on line 10 I recommend changing "student created" to "student-created" as it is a bit confusing as is.